packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -14,14 +14,14 @@ import {
 } from '@deriv/components';
 import { getLegalEntityName, isDesktop, isMobile, routes, validPhone } from '@deriv/shared';
 import { Localize, localize } from '@deriv/translations';
-import FormSubHeader from 'Components/form-sub-header';
-import PoiNameDobExample from 'Assets/ic-poi-name-dob-example.svg';
-import InlineNoteWithIcon from 'Components/inline-note-with-icon';
-import FormBodySection from 'Components/form-body-section';
-import { DateOfBirthField, FormInputField } from 'Components/forms/form-fields';
+import FormSubHeader from '../form-sub-header';
```

Comment:
Changing the type of path is required for building projects with stores hook

Reply:
Alright, let me share the part of explanation received from Farzin, I hope he doesn’t mind it :)

So basically, we build each package in our monorepo (not for api, hooks, stores, shared) and then use the build version of them in other packages.

The problem is when you build a project you bundle everything with it. For example CFD has Store provider and CFD components are using useStore to access that provider data, and when we build CFD, everything get's bundled together.

So now if you use one of built CFD components in another package which is using useStore, you will get an error because store provider doesn't exist for it as it's built. So we are importing the component directly to use the raw version of it.

So that is why we have to change path and import the files directly and why importing from let say @deriv/cfd will not work.

So imported component will be built by the package using it. 

For example a component from CFD that has been used in Appstore and is part of appstore and appstore should build it.

Each package has some aliases for import which is only defined in that package.

Like CFD has it's own aliases, CFD component is using it, Now when you import CFD component in appstore and it becomes part of appstore those aliases don't exist anymore, So it won't work

Reply:
Thanks for the insights. Yet we don't have any issue using the component within the package and I would prefer using alias with the package itself

Reply:
Yup, I think @yauheni-deriv explanation is clear. Let me know if you guys need to jump of a call to discuss it more 🙇🏻 

cc: @matin-deriv